yarn.lock

```diff
@@ -481,6 +491,15 @@
     exenv "^1.2.2"
     prop-types "^15.5.10"
 
+"@atlaskit/theme@^8.0.0":
```

Comment:
I understand that this is Just How Yarn Works, but I'm so sad that we are already getting two versions of theme (at least!) in our project. Won't somebody think of the resolved dependency deduping in yarn?

(This is me kvetching, not an ask for a solution)